04_containers.Rmd

```diff
@@ -701,7 +701,7 @@ We notice that the imported biom file did not contain the sample meta data
 yet, so it includes an empty data frame.
 
 ```{r}
-head(colData(se))
+head(se)
```

Comment:
These are not equivalent, check for isntance 


```
library(mia)
data(enterotype)
se <- enterotype
head(colData(se)) # Operates on the column data of the se object
head(se)  # Operates on the more extensive se object.
```

Reply:
If unclear first familiarize with SummarizedExperiment objects (OMA chapter 3.2)

Reply:
Remember to resolve conversations when done.